package.json

```diff
@@ -71,5 +71,14 @@
         "trailingComma": "all",
         "bracketSpacing": true,
         "arrowParens": "avoid"
+    },
+    "overrides": {
+        "nth-check": "^2.0.1",
+        "package-json": "^8.1.0",
+        "uuid": "^7.0.3"
+    },
+    "engines": {
```

Comment:
There is already an `engines` property, so if you could add these there instead...